model/src/test/kotlin/utils/FindingCurationMatcherTest.kt

```diff
@@ -180,8 +181,9 @@ class FindingCurationMatcherTest : WordSpec() {
                     concludedLicense = "Apache-2.0"
                 )
 
-                val curatedFinding = matcher.apply(finding, curation)!!
+                val curatedFinding = matcher.apply(finding, curation)
 
+                curatedFinding.shouldNotBeNull()
```

Comment:
Would it make sense to move the two checks below also into a lambda, similar to previsous commit?

Reply:
Good point, done.